Source/Events.Processing/Filters/FilterValidators.cs

```diff
@@ -61,7 +61,7 @@ void PopulateFilterValidatorMap()
             {
                 if (TryGetValidatorTypeFor(filterDefinitionType, out var validatorType))
                 {
-                    _logger.Trace($"Filter definition type {filterDefinitionType.FullName} can be validated by validator type {validatorType.FullName}");
+                    _logger.Trace("Filter definition type {FilterFullName} can be validated by validator type {ValidatorFullName}", filterDefinitionType.FullName, validatorType.FullName);
```

Comment:
We should drop the .FullName here
```suggestion
                    _logger.Trace("Filter definition type {FilterType} can be validated by validator type {ValidatorType}", filterDefinitionType, validatorType);
```